p4_symbolic/main.cc

```diff
@@ -75,10 +82,24 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
+  // Parse table entries.
+  p4_symbolic::ir::TableEntries table_entries;
+  if (!entries_path.empty()) {
+    pdpi::StatusOr<p4_symbolic::ir::TableEntries> table_entries_or_status =
+        p4_symbolic::ir::ParseAndFillEntries(entries_path.c_str());
+
+    if (!table_entries_or_status.ok()) {
+      std::cerr << "Could not parse table entries: "
+                << table_entries_or_status.status() << std::endl;
+      return 1;
+    }
+    table_entries = table_entries_or_status.value();
+  }
+
   // Transform to IR and print.
   pdpi::StatusOr<p4_symbolic::ir::P4Program> ir_status =
-      p4_symbolic::ir::Bmv2AndP4infoToIr(bmv2_or_status.value(),
```

Comment:
I also left this comment in the other CL:

create a Status Main() function that uses Status macros, and only do printing and return 1 once in the main

Reply:
I am going to do that in the very last CL to avoid git rebasing conflict issues